crates/aptos/src/common/types.rs

```diff
@@ -1090,6 +1125,18 @@ pub struct MovePackageDir {
     /// Select bytecode, language, compiler for Move 2
     #[clap(long)]
     pub move_2: bool,
+
+    /// Select optimization level
+    #[clap(long, value_parser = clap::value_parser!(OptimizationLevel))]
+    pub optimization_level: Option<OptimizationLevel>,
+
+    /// Enable lint
+    #[clap(long)]
+    pub lint: bool,
```

Comment:
I don't think we want to have a `--lint` option, just the lint command. If we have a lint option we basically encode in the design that every Move command can also lint, which restricts our future design space. Perhaps the linter is a complete different tool than the compiler in the future? The `--lint` flag isn't neede, may confuse users how to use our tools, and creates a restriction for future designs.

Reply:
OK, removing.